tools/preflight/preflight.go

```diff
@@ -800,6 +814,8 @@ func (o *Run) createRestorePodFromSnapshot(ctx context.Context, volSnapshot *uns
 		Create(ctx, restorePVC, metav1.CreateOptions{})
 	if err != nil {
 		o.Logger.Errorln(err.Error())
+		restorePVCYaml := objToYAML(restorePVC)
+		o.Logger.Warnf("restore PVC Creation Failed:\nRestore PVC yaml:\n%s", string(restorePVCYaml))
```

Comment:
A multi-sentence stmt should logged on single line only. Yaml should be printed on newline